### PR TITLE
Fix #4828 When editing attribute mapping for RD-Connect I get a freemarker error

### LINKED ISSUE #4828: StaticContent plugin escapes, and therefor destroys, javascript
#### Reproduce
create static content with javascript

#### Expected
this works

#### Observed
javascript get's escaped and therefor does not work

### PR DESCRIPTION
**Pull request by Mark-de-Haan**
Merge #4825 first

### CODE REVIEW

File: molgenis-data-import/src/main/java/org/molgenis/data/importer/ImportWizardController.java [hunk 1]
```diff
@@ -428,11 +430,13 @@ private ImportRun importFile(HttpServletRequest request, File file, String actio
 	{
 		// no action specified? default is ADD just like the importerPlugin
 		ImportRun importRun;
+		String fileExtension = getExtension(file.getName());
 		DatabaseAction databaseAction = getDatabaseAction(file, action);
-		if (dataService.hasRepository(FilenameUtils.getBaseName(file.getName())))
+		if (fileExtension.contains("vcf") &&
```
Comment (dennishendriksen) [inline molgenis-data-import/src/main/java/org/molgenis/data/importer/ImportWizardController.java]: Other extensions are missing such as vcf.gz

**Comment by fdlk** [comment]
test this please

**Comment by Mark-de-Haan** [comment]
test this please

**Comment by tommydeboer** [comment]
test this please

**Pull request merged**